tests/bagatom/test_bagToXML.py

```diff
@@ -208,3 +231,14 @@ def test_return_tree_with_baggingDate(bagxml_with_bagging_date):
         namespaces={'a': bagatom.BAG_NAMESPACE}
     )
     assert baggingDate[0].text == BAGGING_DATE
+
+
+def test_set_ark_nan(bagxml_with_nondefault_ark):
+    """
+    Check that the ARK NAAN is properly set and that it both can be
+    changed at runtime by a library user and defaults to 67531.
```

Comment:
It doesn't. That's tested elsewhere. Saw the other test and didn't update the comment.